enh(doc): external PR documentation (#8322)
centreon/centreon: CONTRIBUTING.md

You have been working on Centreon base code and want to submit it to us. Well... Again you are more than welcome and thank you in advance ! :clap:

The pull request must comply with certain requirements which are set out in the following template
[PULL_REQUEST_TEMPLATE.md](.github/PULL_REQUEST_TEMPLATE.md)

> > **Notice**: A pull request which contain more than the described modifications or contain more than one issue will

Reviewer: ```suggestion
> **Notice**: A pull request which contains more than the described modifications or contains more than one issue will
```

> be rejected. Please take care of the description and open one pull request per issue

Reviewer: ```suggestion
be rejected. Please take care of the description and open one pull request per issue.
```

> If the PR matches the expected requirements, it will be added to a refinement session with the dev team and the

Reviewer 2: `dev team` -> `development team`

product owner.

> If every is clear enough, it will integrate the dev processes and merged if successfully pass our CI's acceptance tests.

Reviewer: ```suggestion
If everything is clear enough, the PR will integrate the dev processes and will be merged if it successfully passes our CI's acceptance tests
```

> If everything is clear enough, the PR will integrate the dev processes and will be merged if it successfully passes our CI's acceptance tests

Reviewer 2: `CI's` -> `Continous Integration's`

> If everything is clear enough, the PR will integrate the dev processes and will be merged if it successfully passes our CI's acceptance tests

Reviewer 2: `the PR will integrate the dev processes` -> `the PR will integrate our development workflow`

Then our Quality Assurance team will test it, again, to avoid any regression before being able to release it.

> If the dev team need more details, they'll contact you on the pull request. Please stay tuned.

Reviewer: ```suggestion
If the dev team needs more details, they will contact you on the pull request. Please stay tuned.
```

> If the dev team needs more details, they will contact you on the pull request. Please stay tuned.

Reviewer 2: dev team -> development team

> **Warning**: Any pull request that does not respect those requirements will be legitimately rejected ! In addition,

> if you are asked to do so, you must help us understand your changes or problem, and respond to us within 8 days.

Author: ```suggestion
if you are asked to do so, you must help us understand your changes or behavior, and respond to us within 8 days.
```

> **Notice**: We inspired ourselves with another open source project we use, to better communicate on your work :
Visual Studio Code.

> Here are the labels we may add to your work.

Author: ```suggestion
Here are the labels and descriptions we may add to your work.
```

| Label                             | Reason |
| :-----                            | :------ |
| ```pr/external```                 | The first badge |

> | ```status/accepted```             | The dev team have approved your modifications |

Reviewer: ```suggestion
| ```status/accepted```             | The dev team has approved your modifications |
```

> | ```status/implemented```          | The PR have been added or may already be released in next version |

Reviewer: ```suggestion
| ```status/implemented```          | The PR has been added or may have already been released in next version |
```

| ```status/duplicate```            | Another PR is already open |
| ```status/invalid```              | The requirements are not met |

> | ```status/in-backlog```           | The dev team have groomed your proposition and will work on it soon |

Reviewer: ```suggestion
| ```status/in-backlog```           | The dev team has groomed your proposition and will work on it soon |
```

| ```status/could-not-reproduce```  | The dev was unable to reproduce the use-case and may need more information |
| ```status/needs-attention```      | This PR is on hold. The reason is specified in the issue |

> | ```status/needs-merge```          | A dev need to merge your work |

Reviewer: ```suggestion
| ```status/needs-merge```          | A dev needs to merge your work |
```

| ```status/too-dangerous```        | The modification seems to impact too much features or may introduce side effects |

> | ```status/multiple-issues```      | The modifications impact more than one issue in a single PR |

Author: ```suggestion
| ```status/multiple-issues```      | The modifications contains more than one issue in a single PR |
```

> | ```status/multiple-issues```      | The modifications contains more than one issue in a single PR |

Reviewer: ```suggestion
| ```status/multiple-issues```      | The modifications contain more than one issue in a single PR |
```

> | ```status/out-of-scope```         | The issue is out of the described scope |

Author: ```suggestion
| ```status/out-of-scope```         | The modifications are out of the described scope |
```

If you want to visualize and suggest modification to the documentation through a pull request

> you can check the **HOW TO** build the documentation section [here](doc/README.md)

Author: ```suggestion
you can check the **HOW TO** build the documentation section [here](doc/README.md)

```
